### PR TITLE
Solvent manager

### PR DESCRIPTION
**Pull request by SRFU-NN**

### CODE REVIEW

File: src/OptiHPLCHandler/empower_instrument_method.py [hunk 1]
```diff
@@ -131,6 +143,7 @@ class ColumnHandlerMethod(InstrumentMethod):
 
     @property
     def column_temperature(self):
+        """The column temperature."""
```
Comment (SLCB-NN) [inline src/OptiHPLCHandler/empower_instrument_method.py:146]: in sample manager

File: src/OptiHPLCHandler/empower_instrument_method.py [hunk 2]
```diff
@@ -144,6 +157,159 @@ class SampleManagerMethod(ColumnHandlerMethod):
...
+        gradient_table = []
+        for row in self.gradient_data:
+            row_dict = {"Time": row.time, "Flow": row.flow}
+            for solvent, composition in zip(self.solvent_lines, row.composition):
+                row_dict[f"Composition{solvent}"] = composition
+            row_dict["Curve"] = row.curve.value
+            gradient_table.append(row_dict)
+        return gradient_table
+
+    @gradient_table.setter
```
Comment (SLCB-NN) [inline src/OptiHPLCHandler/empower_instrument_method.py:241]: Might get you to explain these next time we meet

**Review by SLCB-NN — CHANGES_REQUESTED** [review]
Just a comment, perhaps we should be very strict with our language in the comments regarding which column oven we are talking about i-e- sample manager column oven / column handler column oven

File: src/OptiHPLCHandler/empower_instrument_method.py (hunk 1 above)
Comment (SRFU-NN) [reply src/OptiHPLCHandler/empower_instrument_method.py:146]: No, this is in the parent method, so it is in both the sample manager and the column manager.

File: src/OptiHPLCHandler/empower_instrument_method.py (hunk 2 above)
Comment (SRFU-NN) [reply src/OptiHPLCHandler/empower_instrument_method.py:241]: Getters and setters?  Sure thing, we will talk about that tomorrow.

**Comment by SRFU-NN** [comment]
> Just a comment, perhaps we should be very strict with our language in the comments regarding which column oven we are talking about i-e- sample manager column oven / column handler column oven

Yeah, as discussed, the parent class for anything with a column oven it going to be renamed to ColumnOvenMethod, and the specific child class for the column manager should then be called ColumnManagerMethod.

**Comment by SRFU-NN** [comment]
I've pushed my changes. We can't support both "manual" changes to the gradient table and changes though the gradient_table property, so for now, I am overwriting the manual ones. Let's discuss tomorrow.

**Comment by SRFU-NN** [comment]
> I've pushed my changes. We can't support both "manual" changes to the gradient table and changes though the gradient_table property, so for now, I am overwriting the manual ones. Let's discuss tomorrow.

I decided that this was too much of a problem, so I refactored the code so the xml and change list act as a combined single source of truth. In a way, it makes the solvent manager class simpler. but let's go through it together.

**Pull request merged**